packages/fabric8-ui/src/app/space/app-launcher/services/app-launcher-gitprovider.service.spec.ts

```diff
@@ -119,4 +119,18 @@ describe('Service: AppLauncherGitproviderService', () => {
         done();
       });
   }));
+
+  it('should create correct build tool detector api url', () => {
+    const correctUrl =
+      'https://detector.api.prod-preview.openshift.io/api/detect/build/https%3A%2F%2Fgithub.com%2Fvikram-raj%2Fapp-dec-21';
```

Comment:
since you are just comparing how a URL is built can you just use `www.example.com`

Reply:
@joshuawilson Done.